OWNERS.md

```diff
@@ -139,6 +145,7 @@ Scope: Tooling for frontend and backend schema-first OpenAPI development.
 | Patrick Jungermann             | Bonial International GmbH | [pjungermann](https://github.com/pjungermann)         | `pjungermann#6933`             |
 | Phil Kuang                     | FactSet Research Systems  | [kuangp](https://github.com/kuangp)                   | `pkuang#3202`                  |
 | Taras Mankovski                | Frontside                 | [taras](https://github.com/taras)                     | `tarasm#1256`                  |
+| Vojta Burian                   | Spotify                   | [sankaritan](https://github.com/sankaritan)           | `vojtabu`                      |
```

Comment:
These would go through a different process, see https://github.com/backstage/backstage/blob/master/GOVERNANCE.md#becoming-an-organization-member

Reply:
Ah ok, I've removed Vojta and Daniel from the organization table, keeping only the project area maintainers in this PR.